cdap-ui/app/cdap/services/WizardStores/AddNamespace/AddNamespaceStore.js

```diff
@@ -17,6 +17,7 @@
 import {combineReducers, createStore, compose} from 'redux';
 import AddNamespaceActions from 'services/WizardStores/AddNamespace/AddNamespaceActions';
 import AddNamespaceWizardConfig from 'services/WizardConfigs/AddNamespaceWizardConfig';
+import {convertMapToKeyValuePairsObj} from 'components/KeyValuePairs/KeyValueStoreActions';
```

Comment:
nit: May be we should extract tis to `services/helpers`. We seem to be using this in a lot of places outside of `KeyValuePairs`

Reply:
This conversion still has a lot of specific behavior for KeyValuePairs component, such as the unique id and also if it's empty, we want to set a default. So I don't think it should sit in helpers.

Reply:
May be extract the functionality of converting a map to key, value? May be the function can accept another argument to add unique ids to each key value (since when we render key-value anywhere in the DOM we need a unique id)? 
` we want to set a default` this is the part that becomes a little out of place for me. We seem to be using a default state of a component inside the store of another component. May be I am missing something.

Reply:
Have extracted out the functionality of converting a map to a key-value pairs array.